android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -115,6 +124,21 @@ public Snapshot(View view) {
     }
     originXByParent = view.getLeft();
     originYByParent = view.getTop();
+    if (view.getBackground() != null) {
+      borderRadius = ((ReactViewBackgroundDrawable) view.getBackground()).getFullBorderRadius();
+    } else if (view instanceof ReactImageView) {
+      borderRadius = getImageBorderRadius((ReactImageView) view);
+    }
+  }
+
+  private float getImageBorderRadius(ReactImageView imageView) {
+    try {
+      Field field = imageView.getClass().getDeclaredField("mBorderRadius");
+      field.setAccessible(true);
+      return field.getFloat(imageView);
+    } catch (NullPointerException | NoSuchFieldException | IllegalAccessException e) {
+      return 0;
```

Comment:
It makes sense to return 0 here instead of throwing some exception but can we add a comment here that explains it explicitly?

Reply:
Resolved in: https://github.com/software-mansion/react-native-reanimated/pull/4103/commits/1a43a54b73c46d1e9a2686143e4c051e75af6191